DigitalLearningSolutions.Data/Models/Register/RegistrationModel.cs

```diff
@@ -30,7 +30,7 @@ string passwordHash
 
         public int JobGroup { get; set; }
 
-        public string PasswordHash { get; set; }
+        public string? PasswordHash { get; set; }

```

Comment:
I think this change has caused a few lines in the RegistrationService to display some IDE warnings complaining about potentially null values. Might be worth going through that and supressing the warnings.